design-documents/graph-ql/coverage/b2b/negotiableQuotes.graphqls

```diff
@@ -230,14 +230,15 @@ type NegotiableQuoteHistoryEntry {
     changes: NegotiableQuoteHistoryChanges
 }
 
-type CartPrices {
-    initial_total_price: Money
+type NegotiableQuotePrices implements QuotePricesInterface {
+    initial_grand_total: Money!
 }
 
-type CartItemPrices {
-    initial_price: Money
-    initial_row_total: Money
-    initial_row_total_including_tax: Money
+type NegotiableQuoteItemPrices implements QuoteItemPricesInterface {
```

Comment:
The type used in the NegotiableQuote schema needs to be updated

Reply:
modified CartItemInterface.itemPrices and deprecated CartItemInterface.prices

Reply:
modified CartItemInterface.prices to return QuoteItemPricesInterface
https://github.com/magento/architecture/pull/489/files#diff-617018aca0d51ab667806b0ca39cd3786df85bd6d6c888641971a42c0b6f16fdR363